src/components/index.d.ts

```diff
@@ -328,7 +328,14 @@ export declare const Button: {
   Unstyled: (downstreamProps: downstreamButtonProps) => any
   WhiteCTA: (downstreamProps: downstreamButtonProps) => any
 }
-
+export declare const PreloadTags: ({
+  crop, publicId, height, width,
+}: {
+  crop: string,
```

Comment:
```suggestion
  crop: 'fill' | 'fit' | 'crop' ,
```
https://github.com/getethos/ethos-design-system/blob/18333e4a7faab047a736bc04c2e0c5e7f0ae553e/src/components/Images/Images.js#L202-L206